app/models/services.py

```diff
@@ -12,38 +13,40 @@ class ErrInvalidParameters(Exception):
     pass
 
 
-# DB initializtion
-db = tinydb.TinyDB("db.json")
-services = db.table("services")
-
-
-def insert(shop_id="", name="", description="", images=None):
-    if not images:
-        images = []
-
+def insert(shop_id="", name="", description="", price="", images=None):
     # Initialize id with randomly generated string
-    service_id = util.random_id(initial="s")
+    id = util.random_id(initial="s")
 
     services.insert(
         {
-            "service_id": service_id,
             "shop_id": shop_id,
+            "id": id,
             "name": name,
             "slug": slugify(name),
             "description": description,
-            "images": images,
+            "price": price,
+            "images": images if images is not None else [],
```

Comment:
we can make this one more idiomatic by doing this: 
```suggestion
            "images": images or [],
```